Makefile

```diff
@@ -12,183 +9,50 @@ DATE_FMT = +%Y%m%d%H%M.%S
 # Fallback mechanism to support other systems:
 # 1. 'date -d': Busybox and GNU coreutils.
 # 2. 'date -r': BSD date. It does not support '-d'.
-BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+export BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+
 
-GO_BUILDFLAGS ?= -trimpath -ldflags="-s -w"
+clean:
+	@rm -rf dist/
+	@docker image ls "$(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME)*" -aq | xargs docker rmi --force
```

Comment:
**issue**: this will fail if there is no docker image

Reply:
I tried this:

```shell
make clean
make clean
echo $?
0
```

and I get the exit code 0. can you elaborate? maybe I'm missing something.

Reply:
mmh

```
$ make clean 
"docker rmi" requires at least 1 argument.
See 'docker rmi --help'.

Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]

Remove one or more images
make: *** [Makefile:11: clean] Error 123
```

Are you perhaps on macOS ? FreeBSD xargs ignores empty args if I recall correctly

Reply:
Oh, yes I'm on mac. Let me try to figure out on how to fix this.

Reply:
Alright, this should do the trick: https://github.com/elastic/apm-aws-lambda/pull/362/commits/d1b6db3aa0903759547e4d042d951c28e4aba315

Reply:
Works like a charm!